src/content/tutorials/observability/data-ingestion/logs/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Ingest logs into the Observability Platform
+linkTitle: Logs
+description: How to make new logs available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-logs
+weight: 50
+last_review_date: 2024-10-09
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How to ingest new logs?
+  - Why do my clusters run Alloy?
+---
+
+By default, Giant Swarm clusters starting from release v29.0.0 are equipped with [Alloy](https://grafana.com/docs/alloy), it's a lightweight, high-performance log agent that collects system logs from your cluster and forwards them to a central [Loki](https://grafana.com/docs/loki) instance running on the management cluster.
+
+The Giant Swarm Observability Platform allows you to ingest logs from your workloads in a self-service way using labels to select which pods to ingest logs from.
+
+### Label your resources
+
+The label `giantswarm.io/logging` can be set to `true` or `false` on any Namespace or Pod. When the label isn't set it's considered `false`.
+
+- Pods in any namespace can be labeled
+- Namespace can be labeled, the value used for the namespace will be the default for all pods in this namespace
+- Pods label take precedence over Namespace label. For example logging can be enabled for a pod in a namespace where logging is disabled and vice versa.
+
+||Pod `giantswar.io/logging=true`|Pod `giantswar.io/logging=false`|Pod `unset`
+|-|------------------------------|--------------------------------|--------------------------
+**Namespace** `giantswar.io/logging=true`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-check"></i>
+**Namespace** `giantswar.io/logging=false`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-times"></i>
+**Namespace** `unset`|<i class="fas fa-check"></i>|<i class="fas fa-times"></i>|<i class="fas fa-times"></i>
+
+You can use the following command to label a namespace or a pod:
+
+```bash
+kubectl label namespace <namespace> giantswarm.io/logging=true
+kubectl label pod <pod> giantswarm.io/logging=true
+```
+
+Logs lines can then be viewed in Grafana Explore, learn more about this in [Exploring logs with LogQL]({{< relref "/tutorials/observability/data-exploration/exploring-logs" >}}).
```

Comment:
Should we explain what labels are added automatically and so on?

Reply:
I have no opinion on this, but if we do it here we should also do it on the metrics page then.

Reply:
Oh yeah that's a good idea. Like

> Per default log ingestion is enabled for the namespaces   
> \- kubesystem  
> \- giantswarm

or something like that?   
Maybe we also need a disclaimer with   
\> These labels should not be overwritten or we cannot guarantee the operational safety of the clusters

something like that? wdyt?

Reply:
Added a section to describe default namespaces and our PodLogs resources that should not be overwritten

Reply:
We agreed to go with PodLogs only.